frappe/utils/data.py

```diff
@@ -208,15 +208,21 @@ def get_datetime_str(datetime_obj):
 		datetime_obj = get_datetime(datetime_obj)
 	return datetime_obj.strftime(DATETIME_FORMAT)
 
-def get_user_format():
```

Comment:
Let's not remove this old method for backwards compatibility.

Reply:
`get_user_format` reinstated (with comment warning) as alias for get_user_date_format.